Assets/Scripts/Layers/LayerTypes/LayerData.cs

```diff
@@ -76,7 +77,20 @@ public bool ActiveInHierarchy
             }
         }
 
-        [JsonIgnore] public List<LayerPropertyData> LayerProperties => layerProperties;
+        [JsonIgnore] public List<LayerPropertyData> LayerProperties
+        {
+            get
+            {
+                // When unserializing, and the layerproperties ain't there: make sure we have a valid list object.
+                if (layerProperties == null)
```

Comment:
misschien beter `layerProperties=new();` doen als default bij de declaratie? dat scheelt weer een check

Reply:
Die staat daar ook, maar ik denk te hebben gemerkt dat na unserializen dat initialisatie overschreven wordt als een property afwezig is in de JSON.. maar ik kan nog een keertje testen want ik zou er ook niet rouwig om zijn als die null-check niet nodig blijkt te zijn